ascii-to-mkdown.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+
+# Convert asciidoc to markdown
+for f in $(find ./documents -type f -name "*.txt"); do
+  MDFILE=$(echo $f | sed "s/.txt/.markdown/g")
+  XMLFILE=$(echo $f | sed "s/.txt/.xml/g")
+  asciidoc -b docbook $f
+  pandoc -f docbook -t markdown_strict $f | sed "s/txt/xml/g" | iconv -f utf-8 > $MDFILE
```

Comment:
Something is adding a / to the beginning of the markdown (e.g. Headings symbols such as # ##) so the headings are not showing..